content/en/docs/release-oversight/risk-analysis.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Risk Analysis"
+description: Overview of Test Failure Risk Analysis.
+---
+
+### Summary
+In an effort to raise awareness for potential regressions, 
+tooling to analyze the overall risk for individual test failures has been created.  Risk 
+Analysis compares failed tests against the historical results of similar periodics and 
+ranks the 'Risk of Regression' based on the previous results. It **does not** analyze the code changed in a PR or payload. It 
```

Comment:
The "does not" analyze the code changed in a PR or payload sentence has me a little confused. Arguably it does, but I suspect you're intending to say it's not like doing code analysis looking for bugs?

Reply:
We analyze the test pass rates to determine risk.  We have no idea what code changed or how it might impact a test.  That was my point in calling that out.

Reply:
If this doesn't feel right to try to make this point / distinction I can remove it.